css/src/tokens/colors.scss

```diff
@@ -16,6 +18,9 @@ $box-shadow-color-extra-heavy: var(--theme-box-shadow-extra-heavy);
 
 $body-background: var(--theme-body-background);
 $body-background-medium: var(--theme-body-background-medium);
+$body-background-learn: var(--theme-body-background-learn);
```

Comment:
remember to update here after the accent update

Reply:
Updated